Formula/butterfly-cli-package.rb

```diff
@@ -0,0 +1,24 @@
+class ButterflyCliPackage < Formula
+  desc "Application code transformation tool"
```

Comment:
Can we change this to `Butterfly application code transformation tool`

Reply:
That's not the homebrew style, it automatically adds that. See https://github.com/Homebrew/homebrew-core/blob/master/Formula/tomcat.rb for example.

Reply:
Got it, thanks.